stdlib/format.ml

```diff
@@ -649,6 +649,8 @@ let pp_print_bool state b = pp_print_string state (string_of_bool b)
 let pp_print_char state c =
   pp_print_as state 1 (String.make 1 c)
 
+let pp_print_nothing _fmt () = ()
```

Comment:
clearly the parameter should be named `_state` for consistency with the code around.